### PR TITLE
Alp 25

### PR DESCRIPTION
**Pull request by sawamotokai**
Removed change in App.js

### CODE REVIEW

File: aunt-leahs-app/src/App.js [hunk 1]
```diff
@@ -2,13 +2,12 @@ import React from 'react';
...
 
-
 function App() {
-  return (
-    <div className='testtest'>
-        <AdminLogin />
-    </div>
-  );
+	return (
+		<div className="testtest">
```
Comment (codygagnon) [inline aunt-leahs-app/src/App.js]: Did you mean to commit this change? I think that the className could be improved.

File: aunt-leahs-app/src/components/header/SubText.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../styles.css'
+
+const SubText =(props)=>{
```
Comment (codygagnon) [inline aunt-leahs-app/src/components/header/SubText.jsx]:
```suggestion
const SubText = (props) => {
```

File: aunt-leahs-app/src/components/header/header.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        <div className='headerBox'>
+                <img className="headerImage header"  alt="Aunt Leah's Logo" src={require('../../AuntLeahsTrees.png')}/>
+                <div className="headerMain header">{headerText.headTwo}</div>
+                {(page !== null) 
+                    &&<SubText title={page}/>}
+        </div>
+    )
+};
+
+export default Head;
```
Comment (codygagnon) [inline aunt-leahs-app/src/components/header/header.jsx]:
```suggestion
export default Head;

```

File: aunt-leahs-app/src/components/locationTable/locationTable.jsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        var updatedLocations = this.state.locations.slice(0);
+        updatedLocations.forEach(l => {
+            if(l.id === id) {
+                l.selected = true;
+                this.props.onLocationSelect(l);
+            } 
+            else {
+                l.selected = false;
+            }
+        });
```
Comment (codygagnon) [inline aunt-leahs-app/src/components/locationTable/locationTable.jsx]: Please name variables more clearly to improve readability.

File: aunt-leahs-app/src/components/locationTable/locationTable.jsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+                    </TableCell>
+                </TableRow>
+            ))}
+            </TableBody>
+        </Table>
+    }
+
+}
+
+export default LocationSelection;
```
Comment (codygagnon) [inline aunt-leahs-app/src/components/locationTable/locationTable.jsx]:
```suggestion
export default LocationSelection;

```

File: aunt-leahs-app/src/components/locationTable/locationTable.jsx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+        this.setState({locations: updatedLocations});
+    }
+
+    render() {
+        return <Table aria-label="simple table">
+            <TableBody>
+            {this.state.locations.map(location => (
+                <TableRow key={location.id} 
+                    onClick={(e) => this.selectLocation(location.id)}
+                    hover={true}
```
Comment (codygagnon) [inline aunt-leahs-app/src/components/locationTable/locationTable.jsx]: For boolean props that are always true, you should be able to simply write the name of the prop, as below.
```suggestion
                    hover
```

File: aunt-leahs-app/src/pages/homepage/homepage.jsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+        </div>
+        
+        <div className="homepage-admin-login">
+            <a href='#'>Admin Login</a>
+        </div>
+        
+    </div>
+);
+
+export default HomePage;
```
Comment (codygagnon) [inline aunt-leahs-app/src/pages/homepage/homepage.jsx]:
```suggestion
export default HomePage;

```

**Review by codygagnon — CHANGES_REQUESTED** [review]
Kai, please have a look at my comments. There are a few small changes that I think should be made. Also, please remember to add a descriptive title to your PRs, so that your reviewers know what you've been working on.

**Comment by codygagnon** [comment]
The recently merged PRs have caused some conflicts. Can someone resolve these?

File: aunt-leahs-app/src/components/locationTable/locationTable.jsx (hunk 6 above)
Comment (sawamotokai) [reply aunt-leahs-app/src/components/locationTable/locationTable.jsx]: I don't know what happened but I didn't modify any codes besides shiftDataPage.
So I don't know if I should make those changes.

**Comment by thaingocnguyen** [comment]
Because the volunteer data page is on this branch as well, I think we should group ALP-25 and ALP-26 into this branch.

File: aunt-leahs-app/src/components/locationTable/locationTable.jsx (hunk 4 above)
Comment (thaingocnguyen) [reply aunt-leahs-app/src/components/locationTable/locationTable.jsx]: This was due to not merging from master. Not present in the code anymore.

File: aunt-leahs-app/src/App.js (hunk 1 above)
Comment (thaingocnguyen) [reply aunt-leahs-app/src/App.js]: This is not in the code anymore.

File: aunt-leahs-app/src/components/locationTable/locationTable.jsx (hunk 6 above)
Comment (thaingocnguyen) [reply aunt-leahs-app/src/components/locationTable/locationTable.jsx]: This has been taken cared of in another issue and is not part of this PR.

**Review by codygagnon — APPROVED** [review]

**Review by thaingocnguyen — APPROVED** [review]

**Pull request merged**